front/app/modules/commercial/content_builder/admin/components/ContentBuilderSettings/index.tsx

```diff
@@ -56,6 +62,15 @@ const ContentBuilderSettings = () => {
       h="100%"
       background="#ffffff"
     >
+      <StyledCloseIconButton
+        className="builderSettingsClose"
```

Comment:
Is this class name used anywhere?

Reply:
I'm using it in the e2e test (`cy.get('.builderSettingsClose').click();`)

Reply:
Ok, I see - why not name is consistently with the other e2e classes and id-s (with the `e2e` prefix)?